sdk/communication/azure-communication-identity/azure/communication/identity/_communication_identity_client.py

```diff
@@ -78,75 +85,73 @@ def from_connection_string(
 
     @distributed_trace
     def create_user(self, **kwargs):
-        # type: (...) -> CommunicationUserIdentifier
+        # type: (Any) -> CommunicationIdentifier
         """create a single Communication user
 
-        :return: CommunicationUserIdentifier
-        :rtype: ~azure.communication.identity.CommunicationUserIdentifier
+        :return: CommunicationIdentifier
+        :rtype: ~azure.communication.identity.CommunicationIdentifier
         """
         return self._identity_service_client.communication_identity.create(
-            cls=lambda pr, u, e: CommunicationUserIdentifier(u.identity.id),
+            cls=lambda pr, u, e: CommunicationUserIdentifier(u.identity.id, raw_id=u.identity.id),
             **kwargs)
 
     @distributed_trace
     def create_user_and_token(
             self,
-            scopes, # type: List[Union[str, "_model.CommunicationTokenScope"]]
+            scopes, # type: List[Union[str, CommunicationTokenScope]]
             **kwargs # type: Any
         ):
-        # type: (...) -> Tuple[CommunicationUserIdentifier, AccessToken]
-        """create a single Communication user with an identity token.
-        :param scopes:
-            List of scopes to be added to the token.
-        :type scopes: list[str or ~azure.communication.identity.models.CommunicationTokenScope]
-        :return: A tuple of a CommunicationUserIdentifier and a AccessToken.
+        # type: (...) -> Tuple[CommunicationIdentifier, AccessToken]
```

Comment:
This will always return a `CommunicationUserIdentifier`. Are we generalizing to `CommunicationIdentifier` here for cross-library is instance checks?

Reply:
We are generalizing here to ensure future compatibility if this library ever expands to support other Identifier types. If we decide it never will, we can always make the return type narrower later :)

Reply:
This method will always return a `CommunicationUserIdentifier` that is why we made it more specific and called it `create_user`. Future potential types would be added as separate methods. In other languages also for this method we return `CommunicationUser`.